README.md

```diff
@@ -1 +1,97 @@
 # Hello Webserv
+
+## Types of Configuration
+
+Configuration file setup can have 4 types of entry:
+
+- One value, they represent only one value.
+```
+item value;
+```
+
+- Multi value, they represent a list of values separated by `,`.
+```
+item value1,value2,value3,...,valueN;
+```
+
+- One value map, they represent a key value pair.
+```
+item key=value;
+```
+
+- Multi value map, they represent a list of key value pairs separated by `,`.
+```
+item key1=value1,key2=value2,key3=value3,...,keyN=valueN;
+```
+
+## Server Block
+
+The config file can have N server blocks, but they represent the highest level of the config file.   
+Inside of them you can have:
+
+- One value ***port***, above 1023 and below 65536.
+```
+port 4321;
+```
+
+- Multi value ***names***, they will not be checked.
+```
+names hello.com,world.com,42.com;
+```
+
+- One value ***limit***, can represent bytes, kilobytes, megabytes or gigabytes.
+```
+limit 1024b;
+limit 1024KB;
+limit 1024mB;
+```
+
+- Multi value map ***error***, represent error code and error file.
+```
+error 404=foo.html,500=bar.html;
+```
```

Comment:
Eu acho que o error fica dentro de route config, ou to viajando?

Reply:
viajando, o erro é pro server inteiro